### ISSUE TITLE
Employee permission new design

### ISSUE THREAD

**Issue by anupam42** [anupam42/PG-DAPP#170]
![whatsapp image 2016-08-06 at 11 01 58 am](https://cloud.githubusercontent.com/assets/16332264/17454939/006cc9f6-5bc6-11e6-979c-03d9d500ed81.jpeg)
Its a long list of permission better we put them in boxes like above 
Ex: Report has too many fields so to start of with we should them only after clicking on it 
.
options : Better to show the pop-up for items in thta screen list.
- [ ] Create Employee profile

**Comment by AlienHu** [comment]
Create , edit , delete and view permissions are added to all modules , @anupam42  Please take up UI part

**Comment by samurailens** [comment]
@GanuM : please take this up as priority.

**Comment by saichaitanyaeruvaram** [comment]
Make it cleaner.

**Comment by samurailens** [comment]
@GanuM  : Are the permissions ported to MApp? 

@gajananalienhu , @ParshuM : How is the testing for this feature?  We need a report on testing on terminal and mobile app for at least 5 users. Please update here.

**Comment by AlienHu** [comment]
@samurailens yes implementation of permissions is similar in both DAPP & MAPP
